### PR TITLE
Build the js-sdk in the CI script

### PR DESCRIPTION
**Pull request by dbkr**
now it's transpiled

### CODE REVIEW

File: jenkins.sh [hunk 1]
```diff
@@ -12,6 +12,9 @@ set -x
 # install the other dependencies
 npm install
 
+# we may be using a dev branch of js-sdk in which case we need to build it
+(cd node_modules/matrix-js-sdk && npm run build)
```
Comment (Kegsay) [inline jenkins.sh]: It's `npm run compile` as `build` is used for [distribution](https://github.com/matrix-org/matrix-js-sdk/blob/master/package.json#L10).

Comment (Kegsay) [inline jenkins.sh]: Same for the vector-web PR.

Comment (dbkr) [inline jenkins.sh]: Now changed as per https://github.com/matrix-org/matrix-js-sdk/pull/320#discussion_r95642977

**Comment by Kegsay** [comment]
LGTM

**Pull request merged**